nuitka/distutils/bdist_nuitka.py

```diff
@@ -77,78 +141,83 @@ def _build(self, build_lib):
-
-        subprocess.check_call(command, cwd=build_lib)
-
-        for root, _, filenames in os.walk(build_lib):
-            for filename in filenames:
-                fullpath = os.path.join(root, filename)
-
-                if fullpath.lower().endswith((".py", ".pyw", ".pyc", ".pyo")):
-                    os.unlink(fullpath)
-
-        os.chdir(old_dir)
-
-        self.build_lib = build_lib
+        to_builds = self._find_to_build()
+        for to_build in to_builds:
+            package, main_filename, finding = findModule(
+                importing=None,
+                module_name=ModuleName(to_build.module_name),
+                parent_package=None,
+                level=0,
+                warn=False,
+            )
+
+            # Check expectations, e.g. do not compile built-in modules.
+            assert finding == "absolute", finding
+
+            if package is not None:
+                output_dir = os.path.join(build_lib, package)
+            else:
+                output_dir = build_lib
+
+            command = [
+                sys.executable,
+                "-m",
+                "nuitka",
+                "--module",
+                "--plugin-enable=pylint-warnings",
+                "--output-dir=%s" % output_dir,
+                "--nofollow-import-to=*.tests",
+                "--show-modules",
+                "--remove-output",
+            ]
+
+            if type(to_build) is py_package:
+                command += [
+                    "--include-package=%s" % package_name.replace("/", ".")
+                    for package_name in to_build.related_packages
+                ]
+
+            else:  # type(to_build) is py_module
+                command += [
+                    "--include-module=%s" % module_name
+                    for module_name in to_build.related_modules
+                ]
+
+            # Process any extra options from setuptools
+            if "nuitka" in self.distribution.command_options:
+                for option, value in self.distribution.command_options[
+                    "nuitka"
+                ].items():
+                    option = "--" + option.lstrip("-")
+                    if value is None:
+                        command.append(option)
+                    elif isinstance(value, bool):
+                        option = "--" + ("no" if not value else "") + option.lstrip("-")
+                        command.append(option)
+                    elif isinstance(value, Iterable) and not isinstance(
+                        value, (unicode, bytes, str)
+                    ):
+                        for val in value:
+                            command.append("%s=%s" % (option, val))
+                    else:
+                        command.append("%s=%s" % (option, value))
+
+            command.append(main_filename)
+
+            # added for clarity
+            my_print("Nuitka:INFO:Building: %s" % to_build, style="yellow")
```

Comment:
Uh, looks like we are imitating the log package, not sure about that at all. Doesn't the color suffice?